src/test/java/edu/cornell/kfs/module/purap/businessobject/TestableRequisitionAccount.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * The Kuali Financial System, a comprehensive financial management system for higher education.
+ *
+ * Copyright 2005-2017 Kuali, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package edu.cornell.kfs.module.purap.businessobject;
+
+import org.kuali.kfs.module.purap.businessobject.PurApAccountingLineBase;
+import org.kuali.kfs.module.purap.businessobject.RequisitionItem;
+
+import java.math.BigDecimal;
+
+/**
+ * Version of RequisitionAccount specifically for testing since it
+ * avoids the Spring call in getAccountLinePercent.
+ */
+public class TestableRequisitionAccount extends PurApAccountingLineBase {
```

Comment:
Can you please clarify why this extra class is needed? Based on the existing code in PurApAccountingLineBase, I'm not seeing why the getAccountLinePercent() method needs to be overridden since it doesn't appear to be making a Spring call.